frontend/src/component/changeRequest/ChangeRequestSidebar/ChangeRequestSidebar.tsx

```diff
@@ -76,15 +77,18 @@ export const ChangeRequestSidebar: VFC<IChangeRequestSidebarProps> = ({
         loading,
         refetch: refetchChangeRequest,
     } = usePendingChangeRequests(project);
-    const { changeState, discardDraft } = useChangeRequestApi();
+    const { discardDraft } = useChangeRequestApi();
     const { setToastApiError } = useToast();
+    const [
+        changeRequestChangesWillOverwrite,
+        setChangeRequestChangesWillOverwrite,
+    ] = useState(false);
 
-    const onReview = async (draftId: number, comment?: string) => {
+    const onReview = async (
+        changeState: (project: string) => Promise<void>,
+    ) => {
         try {
-            await changeState(project, draftId, 'Draft', {
-                state: 'In review',
-                comment,
-            });
+            await changeState(project);
```

Comment:
This is a trick: we need to call the `useChangeRequestApi` hook from a component that's within the provider context. When we call it here, it won't get the right state. As such, we instead move the hook call into the child component, keeping the rest of the logic here.